.agents/workflows/cve-fix.md

```diff
@@ -0,0 +1,231 @@
+#### Fixing CVEs
+
+**Before starting, user must:**
+
+Install/update grype:
+```bash
+grype version || curl -sSfL https://get.anchore.io/grype | sudo sh -s -- -b /usr/local/bin
+```
+
+Fetch latest changes:
+```bash
+git fetch
+```
+
+##### 0. Update Vulnerability Database
+
+```bash
+grype db update
+```
+
+##### 1. Update Branch and Create Fix Branch
+
+```bash
+# Switch to target branch and update to latest
+git checkout release-0.X
+git merge --ff-only origin/release-0.X
+
+# Create fix branch (add -v2, -v3 etc if branch exists; don't delete old versions)
+git checkout -b fix-0.X-cves-YYYY-MM-DD
+```
+
+Replace `0.X`: `release-0.21` → `0.21`, `devel` → `devel`.
+
+One commit per package (may fix multiple CVEs), one PR total.
+
+##### 2. Verify Clean Baseline
+
+```bash
+make pkg/natdiscovery/proto/natdiscovery.pb.go
+git add -u
+git diff -I'^//' --staged
+```
```

Comment:
This isn't related to CVE's and doesn't seem necessary to explain here. This would only be relevant if someone actually changed _natdiscovery.pb.go_ or the protobuf dependency was bumped. If you want to keep it then it would be useful to explain the step and why it may be needed.